syncd/RequestShutdownCommandLineOptions.cpp

```diff
@@ -37,7 +37,9 @@ void RequestShutdownCommandLineOptions::setRestartType(
 #define STRING_RESTART_COLD         "COLD"
 #define STRING_RESTART_WARM         "WARM"
 #define STRING_RESTART_FAST         "FAST"
+#define STRING_RESTART_EXPRESS      "EXPRESS"
 #define STRING_RESTART_PRE_SHUTDOWN "PRE-SHUTDOWN"
+#define STRING_RESTART_PRE_EXPRESS_SHUTDOWN "PRE-EXPRESS-SHUTDOWN"
```

Comment:
whats the differnce between express and pre express shutdown ?

Reply:
pre express shutdown is an indication for platform to prepare for express reboot. It is invoked during syncd pre shutdown before reboot similar to PRE-SHUTDOWN. Express is similar to WARM which is invoked after reboot